bulk_grades/api.py

```diff
@@ -150,16 +152,17 @@ def get_rows_to_export(self):
         my_name = self.display_name
 
         students = get_scores(location)
-
-        enrollments = _get_enrollments(location.course_key,
+        course_key = location.course_key
+        enrollments = _get_enrollments(course_key,
                                        track=self.track,
                                        cohort=self.cohort)
         for enrollment in enrollments:
+            cohort = get_cohort(enrollment['user'], course_key, assign=False)
             row = {
                 'block_id': location,
                 'title': my_name,
-                'points': None,
-                'last_points': None,
+                'New Points': None,
+                'Previous Points': None,
```

Comment:
It seems a bit odd to have these be the only ones formatted to be human readable; should we follow through and rename the other columns as well?

Reply:
I agree that it seems odd, but that's what the story says. Maybe the next story will rename the rest of the columns.